### DISCUSSION TITLE
Roam-like control of folding and unfolding through the vertical styling

### DISCUSSION THREAD

**Discussion by jakemkc** [vslinko/obsidian-outliner#223] [Archive]
Hi,

Thanks @vslinko for creating this little wonderful plugin for Obsidian. It works with the Zoom plugin and make things easier.

I would like to ask is it possible to make the vertical styling of the outline clickable for folding/unfolding the list.

A vertical clickable line is much better than a point for this purpose.

Here is an example from Roam: clicking on the left vertical line will fold "testing3" and "testing 4" 
<img width="158" alt="Screen Shot" src="https://user-images.githubusercontent.com/18485734/133148260-f35c7d89-f1aa-40e7-9002-32db1a2c473c.png">

Best
Jake

**Comment by vslinko** [comment]
Released in https://github.com/vslinko/obsidian-outliner/releases/tag/2.1.0
